otter/convergence/gathering.py

```diff
@@ -205,23 +205,28 @@ def _lb_configs_from_metadata(server):
 
     # throw away any value that is malformed
 
-    def parse_config(config, key):
-        if config.get('type') != 'RackConnectV3':
-            lbid = k[len(_key_prefix):]
+    def parse_config(lb_type, lb_id, configs):
```

Comment:
I must admit I was a little confused reading this code, and I'm not sure how to improve it. lb_type is unused, but that's only because RCv3 LBs are manually thrown out further down, right? Are we going to eventually do this for RCv3 too?